### PR TITLE
Adds support for edge and the new dt setup

### PR DESCRIPTION
**Pull request by orta**
I got it working in WSL2, but it doesn't reliably connect to the server over the OS' - so it loads at least,

### CODE REVIEW

File: src/chrome.ts [hunk 1]
```diff
@@ -6,9 +6,9 @@ import Registry = require("winreg");
 import { EventEmitter } from "events";
 
 const defaultChromePaths: Partial<Record<NodeJS.Platform, string[]>> = {
-    win32: ["C:/Program Files/Google/Chrome/Application/chrome.exe", "C:/Program Files (x86)/Google/Chrome/Application/chrome.exe"],
+    win32: ["C:/Program Files/Google/Chrome/Application/chrome.exe", "C:/Program Files (x86)/Google/Chrome/Application/chrome.exe", "C:\Program Files (x86)\Microsoft\Edge Beta\Application\msedge.exe"],
     darwin: ["/Applications/Google Chrome.app/Contents/MacOS/Google Chrome"],
-    linux: ["/usr/bin/google-chrome"],
+    linux: ["/usr/bin/google-chrome", "/mnt/c/Program Files (x86)/Microsoft/Edge Beta/Application/msedge.exe"],
```
Comment (elibarzilay) [inline src/chrome.ts:11]: Should really treat WSL specifically if you want to access Windows stuff from it.  (E.g., there's a `wslpath` that should be used to access Windows paths.)

File: src/github.ts [hunk 2]
```diff
@@ -31,7 +31,7 @@ export interface GetPullFailureResult {
 export type GetPullResult = GetPullSuccessResult | GetPullFailureResult;
 
 export class ProjectService<K extends string = "Check and Merge" | "Review"> {
-    static readonly defaultProject = "Pull Request Status Board";
+    static readonly defaultProject = "New Pull Request Status Board";
     static readonly defaultColumns = ["Check and Merge", "Review"] as const;
```
Comment (elibarzilay) [inline src/github.ts:35]: IIUC, these should change too, right?

Comment (elibarzilay) [reply src/github.ts:35]: Not opening a new PR, but this should also be updated with the new "Needs Maintainer Action" column.

**Comment by orta** [comment]
https://github.com/DefinitelyTyped/focus-dt/commit/19ebc43d7b4b467c35f56ca492312d9cc8a24a3a handles all this 👍🏻

**Pull request closed without merging**